lib/player.rb

```diff
@@ -124,5 +132,24 @@ def cpu_fires_zee_missle
     end
   end
 
+  def find_cells_hit
+    cpu_hits_cells = @board.cells.values.select do |cell|
+      cell if cell.fired_upon? && cell.empty? == false && cell.ship.sunk? == false
```

Comment:
we can probably just do cell if cell.fired_upon? && !cell.empty? &&  !cell.ship.sunk? here